tests/e2e/app/src/Main.js

```diff
@@ -0,0 +1,175 @@
+import React, {useEffect, useMemo, useState} from 'react';
+import {
+    Button, StyleSheet, Text, View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import _ from 'underscore';
+import Onyx, {withOnyx} from 'react-native-onyx';
+import ONYXKEYS from './keys';
+import * as Updates from './lib/updates';
+import * as Data from './data';
+
+const propTypes = {
+    session: PropTypes.shape({login: PropTypes.string}),
+    // eslint-disable-next-line react/forbid-prop-types
+    pokedex: PropTypes.shape({pokemon: PropTypes.arrayOf(PropTypes.object)}),
+    randomNumber: PropTypes.shape({number: PropTypes.number}),
+    allMeteorites: PropTypes.objectOf(PropTypes.arrayOf(PropTypes.object)),
+    allAsteroids: PropTypes.objectOf(PropTypes.arrayOf(PropTypes.object)),
+};
+
+const defaultProps = {
+    session: {},
+    pokedex: {},
+    randomNumber: {},
+    allMeteorites: {},
+    allAsteroids: {},
+};
+
+const styles = StyleSheet.create({
+    container: {
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        rowGap: 20,
+        maxWidth: '100%',
+    },
+    containerButtons: {
+        flexDirection: 'row',
+        justifyContent: 'center',
+        alignItems: 'center',
+        columnGap: 20,
+    },
+});
+
+function Main(props) {
+    const [isLeader, setIsLeader] = useState(
+        Onyx.isClientTheLeader ? Onyx.isClientTheLeader() : false,
```

Comment:
Is it possible that `Onyx.isClientTheLeader` does not exist? AFAIK, it is a noop, but exists.

Reply:
Ah, this is dead code, thank you for pointing it out